### PR TITLE
IGAPP-1122 Hide Chat on Location Route

### PR DESCRIPTION
**Pull request by LeandraH**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

To test:
1. Go to http://localhost:9000/muenchen/de/locations
2. See that there is no chatbot icon
3. http://localhost:9000/muenchen/de/news/local
4. See that there is a chatbot icon

### CODE REVIEW

**Review by steffenkleinle — APPROVED** [review]
Tested on firefox, works as expected.

Not 100% sure but I think I would prefer an optional prop `showChatbot = true` instead of the if check in location layout. I think the power of deciding what to show should be part of the routes responsibilities and not the layouts. What do you think? Other opinions @sarahsporck @SteffiStuffel @f1sh1918?

File: release-notes/unreleased/IGAPP-1122-hide-chatbot-on-maps.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+issue_key: IGAPP-1122
+show_in_stores: true
+platforms:
+  - web
+en: Hide the Icon for the Chatbot on the Locations Route
```
Comment (f1sh1918) [inline release-notes/unreleased/IGAPP-1122-hide-chatbot-on-maps.yml:5]:
```suggestion
en: Hide the icon for the chatbot on the locations route
```

**Review by f1sh1918 — APPROVED** [review]
Tested on chrome :)

**Comment by LeandraH** [comment]
> Tested on firefox, works as expected.
> 
> Not 100% sure but I think I would prefer an optional prop `showChatbot = true` instead of the if check in location layout. I think the power of deciding what to show should be part of the routes responsibilities and not the layouts. What do you think? Other opinions @sarahsporck @SteffiStuffel @f1sh1918?

Hmmm, good question, I'm more inclined to leave it in the layout because the reason for the chatbot being hidden is purely visual, and if we change the design in the future, we can probably also add the chatbot back in. But it's not a hill that I'm willing to die on :D

**Comment by sarahsporck** [comment]
> > Tested on firefox, works as expected.
> > Not 100% sure but I think I would prefer an optional prop `showChatbot = true` instead of the if check in location layout. I think the power of deciding what to show should be part of the routes responsibilities and not the layouts. What do you think? Other opinions @sarahsporck @SteffiStuffel @f1sh1918?
> 
> Hmmm, good question, I'm more inclined to leave it in the layout because the reason for the chatbot being hidden is purely visual, and if we change the design in the future, we can probably also add the chatbot back in. But it's not a hill that I'm willing to die on :D

I think I agree with Leandra with this. Her proposed solution is very simple and not very intrusive. Modifying every route for this seems a little bit overkill when there is no reason to do that right now or in the near future. and if a reason arises at some point, this can be change very easily. No reason to create more work fo this imo. :)

**Pull request merged**